src/library/scala/concurrent/duration/Duration.scala

```diff
@@ -80,7 +80,7 @@ object Duration {
   private[this] val timeUnitLabels = List(
     DAYS         -> "d day",
     HOURS        -> "h hour",
-    MINUTES      -> "min minute",
+    MINUTES      -> "m min minute",
```

Comment:
This would also cause "mins" to be accepted for minutes, which may or may not be something we want.

Reply:
You're right. IMO it could be acceptable as it would make it even more consistent with the others.